### DISCUSSION TITLE
Home page as default?

### DISCUSSION THREAD

**Discussion by gost4711** [music-assistant/support#1469] [Q&A]
hi,

When I start MA, the artist page is always displayed first. How can I display the home page first?

Thanks
GOst4711

**Comment by OzGav** [comment]
The UI is currently in development. Remembering the last displayed page is on the roadmap

**Comment by gost4711** [comment]
OK fine thanks. 
I think it's very important. That's what the USER sees.

**Comment by OzGav** [comment]
This is implemented
